tests/main.test.ts

```diff
@@ -552,19 +544,40 @@ describe('zip-it-and-ship-it', () => {
     },
   )
 
-  testMany('Works with many dependencies', [...allBundleConfigs], async (options) => {
-    const fixtureTmpDir = await tmpName({ prefix: 'zip-it-test' })
-    const opts = merge(options, {
-      basePath: fixtureTmpDir,
+  describe('many-dependencies fixture', () => {
+    let fixtureTmpDir: string
+
+    beforeAll(async () => {
+      fixtureTmpDir = await tmpName({ prefix: 'many-dependencies' })
+      const basePath = `${fixtureTmpDir}/many-dependencies`
+
+      await cpy('many-dependencies/**', basePath, { cwd: FIXTURES_DIR })
+
+      await execa('npm', ['install', '--no-package-lock', '--no-audit', '--prefer-offline', '--progress=false'], {
+        cwd: basePath,
+      })
+    })
+
+    afterAll(async () => {
+      await rm(fixtureTmpDir, { recursive: true, force: true })
     })
 
-    const basePath = `${fixtureTmpDir}/many-dependencies`
-    await cpy('many-dependencies/**', basePath, { cwd: FIXTURES_DIR })
-    await execa('npm', ['install', '--no-package-lock', '--no-audit'], {
-      cwd: basePath,
+    testMany('Works with many dependencies', [...allBundleConfigs], async (options) => {
```

Comment:
This test was for each bundler copying the fixture to a temp directory and running npm install. This was completely unnecessary as `zipNode()` does not actually modify the fixture in any way.

So we do this once before the test and run all tests against this prepared fixture.

Afterwards we also cleanup the folder, which wasn't done before.